### DISCUSSION TITLE
Copy-to-clipboard functionality for code blocks

### DISCUSSION THREAD

**Discussion by robbienohra** [adityatelange/hugo-PaperMod#171] [Ideas]
Not sure what the best approach would be. But it would be pretty sweet if copy-to-clipboard functionality was baked in to code blocks.

**Comment by adityatelange** [comment]
I'll see if we can add this :)

**Reply by liugangjian** [reply]
Hello, have these functions been added? I saw this feature in the demo, but I didn't find it when I applied the theme.

**Reply by shiftingsand** [reply]
I'm wondering this also.  I see "Code Block Copy buttons" in the list of PaperMod features but don't see any documentation on how to enable it.
